app/_hub/kong-inc/prometheus/index.md

```diff
@@ -78,8 +87,12 @@ dashboard: [https://grafana.com/dashboards/7424](https://grafana.com/dashboards/
 - **Connections**: Various Nginx connection metrics like active, reading,
   writing, and number of accepted connections.
 - **Target Health**: The healthiness status (`healthchecks_off`, `healthy`, `unhealthy`, or `dns_error`) of Targets
+<<<<<<< HEAD
   belonging to a given Upstream.
-- **Dataplane Status**: The last seen timestamp, config hash, and config sync status for
+=======
```

Comment:
@kikito I think your git messages ` <<<<<<< HEAD ` were merged here in the doc as text- please remove.

Reply:
that was my fault - sorry @kikito